packages/dx-react-grid-material-ui/package.json

```diff
@@ -86,7 +86,9 @@
     "@devexpress/dx-react-core": "1.0.0-alpha.2",
     "@devexpress/dx-react-grid": "1.0.0-alpha.2",
     "prop-types": "^15.5.8",
-    "react": "^15.5.4"
+    "react": "^15.5.4",
+    "react-tap-event-plugin": "^2.0.1",
+    "enzyme": "^2.8.2"
```

Comment:
`enzyme` is a dev dependency. To disable `eslint` errors in `*.test.*` files see solution in the `.eslintrc.json` file of the `dx-react-core` package